src/applications/lgy/coe/form/validations.js

```diff
@@ -27,9 +27,9 @@ export const validateDocumentDescription = (errors, fileList) => {
 };
 
 export const validateUniqueVALoanNumber = (errors, data) => {
-  const loans = (data || []).map(loan =>
-    replaceNonDigits(loan?.vaLoanNumber || ''),
-  );
+  const loans = (data || [])
+    .map(loan => replaceNonDigits(loan?.vaLoanNumber || ''))
+    .filter(Boolean);
```

Comment:
Out of curiosity, what does this part do?

Reply:
While testing, I added 2 loans, both with the VA loan number blank and I saw a VA loan number must be unique. So the `filter(Boolean)` is removing empty VA loan numbers from the validation check